OutOfSchool/OutOfSchool.DataAccess/Models/EmailOutbox.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+
+namespace OutOfSchool.Services.Models;
+public class EmailOutbox : IKeyedEntity<Guid>
```

Comment:
not sure if it matters much, but this is internal data so we can use `long` ids .